### DISCUSSION TITLE
All Subsets being Imported?

### DISCUSSION THREAD

**Discussion by szarapka** [fontsource/fontsource#620] [Q&A] [answered]
In a Vue3 (ssr) app with tailwind, I'm importing the following:

```
import "@fontsource/open-sans/300.css"
import "@fontsource/open-sans/600.css"
import "@fontsource/open-sans/700.css"
import "@fontsource/inter/100.css"
import "@fontsource/inter/400.css"
import "@fontsource/inter/600.css"
import "@fontsource/inter/700.css"
import "@fontsource/source-serif-pro/700.css"
```

For some reason, this is importing every single subset resulting in 59 font files being included. I'm unsure why this behavior is happening and unsure how to fix it - anyone have any advice/tips here?

When I do the same import in Astro with Tailwind, I get just the latin fonts, as expected.

**Comment by jwr1** [comment, marked as answer]
The `[weight].css` files are supposed to include all subsets of that particular weight  (and is non-italic). To only include the Latin subset of weight 400, for example, you can import `latin-400.css`.

**Reply by szarapka** [reply]
Interesting, I wonder why when using the same imports with [Astro](https://astro.build/) it only pulls just the latin files. This makes sense though and thanks!
